proto/ibc/core/channel/v1/channel.proto

```diff
@@ -79,6 +81,18 @@ enum State {
   STATE_ACKUPGRADE = 7 [(gogoproto.enumvalue_customname) = "ACKUPGRADE"];
 }
 
+// FlushStatus defines the status of a channel end pertaining to in-flight packets.
```

Comment:
nit: should we explicitly mention upgrade handshake? you can technically "flush" packets when you're in an active/open state

```suggestion
// FlushStatus defines the status of a channel end pertaining to in-flight packets during an upgrade handshake.
```